README.md

```diff
@@ -254,44 +252,44 @@ Apache Airflow version life cycle:
 
 Limited support versions will be supported with security and critical bug fix only.
 EOL versions will not get any fixes nor support.
 We always recommend that all users run the latest available minor release for whatever major version is in use.
-We **highly** recommend upgrading to the latest Airflow major release at the earliest convenient time and before EOL date.
+We **highly** recommend upgrading to the latest Airflow major release at the earliest convenient time and before the EOL date.
 
 ## Support for Python and Kubernetes versions
 
-As of Airflow 2.0 we agreed to certain rules we follow for Python and Kubernetes support.
+As of Airflow 2.0, we agreed to certain rules we follow for Python and Kubernetes support.
 They are based on the official release schedule of Python and Kubernetes, nicely summarized in the
 [Python Developer's Guide](https://devguide.python.org/#status-of-python-branches) and
 [Kubernetes version skew policy](https://kubernetes.io/docs/setup/release/version-skew-policy/).
 
 1. We drop support for Python and Kubernetes versions when they reach EOL. We drop support for those
    EOL versions in main right after EOL date, and it is effectively removed when we release the
    first new MINOR (Or MAJOR if there is no new MINOR version) of Airflow
-   For example for Python 3.6 it means that we drop support in main right after 23.12.2021, and the first
+   For example, for Python 3.6 it means that we drop support in main right after 23.12.2021, and the first
    MAJOR or MINOR version of Airflow released after will not have it.
 
 2. The "oldest" supported version of Python/Kubernetes is the default one. "Default" is only meaningful
-   in terms of "smoke tests" in CI PRs which are run using this default version and default reference
+   in terms of "smoke tests" in CI PRs, which are run using this default version and the default reference
    image available. Currently `apache/airflow:latest` and `apache/airflow:2.1.3` images
-   are both Python 3.6 images, however the first MINOR/MAJOR release of Airflow release after 23.12.2021 will
+   are both Python 3.6 images. However, the first MINOR/MAJOR release of Airflow release after 23.12.2021 will
    become Python 3.7 images.
 
 3. We support a new version of Python/Kubernetes in main after they are officially released, as soon as we
    make them work in our CI pipeline (which might not be immediate due to dependencies catching up with
-   new versions of Python mostly) we release a new images/support in Airflow based on the working CI setup.
+   new versions of Python mostly) we release new images/support in Airflow based on the working CI setup.
 
 ### Additional notes on Python version requirements
 
-* Previous version [requires](https://github.com/apache/airflow/issues/8162) at least Python 3.5.3
-  when using Python 3
+* Previous versions [require](https://github.com/apache/airflow/issues/8162) at least Python 3.5.3
```

Comment:
```suggestion
* Airflow 1.10 versions [require](https://github.com/apache/airflow/issues/8162) at least Python 3.5.3
```

Reply:
Your suggestion seems a bit ambiguous. I could interpret it as version 1.10 and newer require Python 3.5.3 or just version 1.10 requires Python 3.5.3+?

Reply:
1.10. Newer versions do not work with 3.5 at all

Reply:
actually we can remove the whole chapter I think

Reply:
ok